apps/studio/components/layouts/ProjectLayout/RestoreFailedState.tsx

```diff
@@ -0,0 +1,41 @@
+import Link from 'next/link'
+import { Button } from 'ui'
+import { CriticalIcon } from 'ui-patterns/Icons/StatusIcons'
+
+import { useProjectContext } from './ProjectContext'
+
+const RestoreFailedState = () => {
+  const { project } = useProjectContext()
+
+  return (
+    <div className="flex items-center justify-center h-full">
```

Comment:
can we also update RestoringState to deprecate the restoring failed state there?
seems like we can now that we have a specific status for when restoring failed